src/pages/Jieun/Login/Login.js

```diff
@@ -1,11 +1,86 @@
-import React from 'react'
-
-const Login = () => {
-    return (
-        <div>
-            
-        </div>
-    )
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './Login.scss';
+
+import '../../../styles/reset.scss';
+import '../../../styles/common.scss';
+
+function Login() {
+  const [id, setId] = useState('');
+  const [pw, setPw] = useState('');
+  const [loginBtn, setLoginBtn] = useState(false);
+
+  const loginBtnChange = () => {
+    return id.includes('@') && pw.length > 4
+      ? setLoginBtn(true)
+      : setLoginBtn(false);
+  };
```

Comment:
위와 연관지어서, 변수로 만들어서 사용
```suggestion
  const isLoginValid = id.includes('@') && pw.length > 4;
```

Reply:
이때, boolean 값을 가지는 변수이기 때문에 앞에 prefix로 is를 붙여 boolean값을 가짐을 나타냄